### ISSUE TITLE
Asterisk logs on remote syslog server not parsed properly

### ISSUE THREAD

**Issue by miken32** [fail2ban/fail2ban#2060]
### Environment:
- Fail2Ban version (including any possible distribution suffixes): 0.9.6.1el6.1
- OS, including release name/version: RHEL 6.9
- [x] Fail2Ban installed via OS/distribution mechanisms
- [x] You have not applied any additional foreign patches to the codebase
- [ ] Some customizations were done to the configuration (provide details below is so)

### The issue:
Asterisk seems to format log entries differently when sending to a remote syslog server. The existing regex for the log line prefix does not capture entries on a remote server.

#### Relevant lines from monitored log files in question:
Local log:
```
[2018-03-01 15:35:53] WARNING[1195][C-00000b43] Ext. s: "Rejecting unknown SIP connection from 67.225.189.160"
```
Remote log:
```
Mar  1 15:35:53 mypbx asterisk[2350]: WARNING[1195][C-00000b43]: Ext. s:6 in @ from-sip-external: "Rejecting unknown SIP connection from 67.225.189.160"
```
#### Recommended fix
Differences are a colon following the call identifier `[C-00000b43]` and a more complete info on the dialplan (`Ext. s:6 in @ from-sip-external` rather than `Ext. s`). This amended regex should account for those differences:

    log_prefix= (?:NOTICE|SECURITY|WARNING)%(__pid_re)s:?(?:\[C-[\da-f]*\])?:? [^:]+:\d*(?:(?: in)? [^:]+:)?

**Comment by sebres** [comment]
Crystal clear and exemplary description, issue fixed now in 8c291cad384cfe62234246f4a4eebf6bfcec2594.
Thanks!

**Comment by miken32** [comment]
Glad to help. Will this get backported to older branches as well? (I'm stuck with a bunch of RHEL 6 installs on 0.9.x)
